shared/reducers/tracker.js

```diff
@@ -267,9 +271,10 @@ function updateUserState (state: TrackerState, action: Action): TrackerState {
       }
 
     case Constants.reportLastTrack:
+      const lastTrack = action.payload && action.payload.track
       return {
         ...state,
-        lastTrack: action.payload && action.payload.track,
+        currentlyFollowing: !!lastTrack,
```

Comment:
we only used currentlyFollowing, this way we don't need lastTrack in the store at all